### PR TITLE
Third branch

### PR DESCRIPTION
**Pull request by LC-GHub**
I gonna merge to staging. Hue Hue.

### CODE REVIEW

File: newfile.txt [hunk 1]
```diff
@@ -0,0 +1,13 @@
+new file
+asdasdasd
+asdasdasdasda
+asdasdasdadasdasda
+asdasdassssssssssssssssss
+sssssssssssssssssssssssssssssssss
+sssssssssssssssssssssssssssssssssssss
+
```
Comment (xiaopang254) [inline newfile.txt]: please remove empty lines

File: baka.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+hello there baka
```
Comment (xiaopang254) [inline baka.txt]: please put in next line for "baka"

**Review by xiaopang254 — CHANGES_REQUESTED** [review]
Lionel, please resolve the issues stated

File: baka.txt (hunk 2 above)
Comment (LC-GHub) [reply baka.txt]: done.

File: newfile.txt (hunk 1 above)
Comment (LC-GHub) [reply newfile.txt]: done.

Comment (xiaopang254) [reply newfile.txt]: ok resolved

**Review by xiaopang254 — APPROVED** [review]
approved

**Review by xiaopang254 — CHANGES_REQUESTED** [review]
don't merge to main

**Pull request closed without merging**